frontend/public/components/_masthead.scss

```diff
@@ -1,3 +1,56 @@
+.co-app-launcher {
+  ul {
+    list-style: none;
+    margin-bottom: 0;
+    padding-left: 0;
+  }
+
+  .pf-c-app-launcher__menu-item-external-icon {
+    --pf-c-app-launcher__menu-item-external-icon--Color: var(--pf-global--icon--Color--light);
+
+    opacity: 1;
+  }
+}
+
+.co-cluster-menu {
+  max-width: 80vw;
+
+  @media (min-width: $screen-sm) {
+    max-width: 90vw;
+  }
+}
+
+.co-cluster-selector {
+  display: inline-grid !important;
```

Comment:
```suggestion
  background-color: transparent !important;
  display: inline-grid !important;
```

Reply:
Not needed since we're not removing `variant="plain"`.